pkg/repo/sysmeta.go

```diff
@@ -77,13 +90,13 @@ func (fsr *FsRepo) ReadLastUpdateCheck() (time.Time, error) {
 
 // WriteLastUpdateCheck updates the LastUpdateCheck in the metadata.
 // It fails if the metadata file doesn't exist.
-func (fsr *FsRepo) WriteLastUpdateCheck(lastUpdateCheck time.Time) error {
+func (fsr *MetadataStore) WriteLastUpdateCheck(lastUpdateCheck time.Time) error {
 	return fsr.updateExistingMetadata(func(m *SystemMetadata) {
 		m.LastUpdateCheck = lastUpdateCheck
 	})
 }
 
-func (fsr *FsRepo) ReadInstallationID() (string, error) {
+func (fsr *MetadataStore) ReadInstallationID() (string, error) {
 	sysmeta, err := fsr.readMetadata()
 	if err != nil {
 		return "", err
```

Comment:
installationID should no longer be in the meta store. It is fine to keep the method here if you prefer, but the source is not `system_metadata.yaml`

Reply:
Keeping it in the store prevents it from being scattered in different files. This was the design we decided here: https://www.notion.so/expanso/Rethinking-Configuration-435fbe87419148b4bbc5119d413786eb?pvs=4#5d5f7298b9ef4ab4bcb1fd6a44ca8c30. Additionally, for users migrating from v1.4.0 to v1.5.0, the installationID will be moved to the metadata store from its previous location the config file.

I know we discussed this in our meeting; How would you like to proceed here?
We can continuing doing what we do now and keep it in the store, or we can modify the metadata store to check both locations - its internal file and the file produced by the installation script. The latter feels like unnecessary complexity and coupling with the implementation of the install script.

Reply:
We know the installationID is not persisted in v1.4.0 https://github.com/bacalhau-project/bacalhau/issues/4085. The discussion to set the installationID outside of repo happened few weeks ago after the design doc, and that why you already changed the install script to write to the new location.

Lets please continue with the implementation to just check the new location

Reply:
iirc https://github.com/bacalhau-project/bacalhau/issues/4085 was about adding the installationID to the job metadata, not persisting it to the filesysystem

Reply:
Does this mean we should change the migration here? https://github.com/bacalhau-project/bacalhau/blob/main/pkg/repo/migrations/v3_4.go#L57. i.e. where do we put an installationID for users that already have on in their config?

Reply:
I have removed the installationID bits from the migration